src/styles/flex.scss

```diff
@@ -0,0 +1,233 @@
+@import './new-settings';
```

Comment:
shouldn't be common CSS in a different folder? in case we will have 10+ common stylesheets there will be a mess.

Reply:
yes, it should be! It's even with a different prefix: `sap-` not `fd-`. I will open soon a new PR for `display` and after it's merged I will rebase main and move this in the proper folder.